### ISSUE TITLE
Accessibility issues in color picker

### ISSUE THREAD

**Issue by uipath-utkarsh** [zefoy/ngx-color-picker#360]
Hi,
Our accessibility team raised a few accessibility issues with the ngx-color-picker which are not yet fixed in v17.

Below is the summary of the bugs

The up/down arrow button is not accessible via keyboard.  Cannot move to this button by pressing tab. 
![image](https://github.com/user-attachments/assets/4a4e960b-f2f6-4d83-91c3-85e7851f9d75)

The up/down arrow button does not have any aria-label/accessible name.
![image](https://github.com/user-attachments/assets/6cac9204-1551-446f-b50a-b50a1b00a7b4)

The input boxes do not have an accessible name/aria label. 
![image](https://github.com/user-attachments/assets/72b0d2ab-7a1b-4acc-b8f6-71e860d02b93)

Can we know if these issues are already fixed in upcoming version? 

Thanks and Best Regards.

**Comment by SuphalAthlur** [comment]
Adding on to this issue,

The spin button to switch between color formats does not have an accessible role and accessible label

![image](https://github.com/user-attachments/assets/e0327ea5-4f2e-4db1-9379-4fce8a95b00a)
